compiler/ballerina-parser/src/main/java/io/ballerinalang/compiler/internal/parser/incremental/HybridNodeStorage.java

```diff
@@ -178,4 +178,8 @@ private TextEditRange markAffectedRange(ModulePartNode oldTree, TextEdit textEdi
                 (textRange.startOffset() - affectedTokenStartOffset);
         return new TextEditRange(affectedTokenStartOffset, textRange.endOffset(), newTextLength);
     }
+
+    public int getCurrentIndex() {
```

Comment:
Looks like other methods don't have the public qualifier. Shall we do the same?
Also, let move this to the other non-private methods section.

Reply:
Done